release.sh

```diff
@@ -33,15 +33,9 @@ IFS=$'\n'
 #   6. If current branch is train branch, pull from origin.
 #   7. Otherwise checkout existing train branch or create fresh one from main.
 #   8. For each of the "main" packages...
-#      8.1. List commits since the last tag.
-#      8.2. For each commit...
-#           8.2.1. Add the commit message to a summary string.
-#      8.3. If CHANGELOG.md exists, write the summary string to CHANGELOG.md.
 #      8.4. If package.json exists, update the version string in package.json.
 #      8.5. If package-lock.json exists, update the version string in package-lock.json.
 #      8.6. If npm-shrinkwrap.json exists, update the version string in npm-shrinkwrap.json.
```

Comment:
nit: do we want to change these to 8.1, 8.2, 8.3 now that we removed the first few items?
(and I guess if so, we'd need to change the comments below [L143, L149, and L155] to match as well)

Although we could potentially argue whether 8.5 and 8.6 are needed anymore since I don't think we use "package-lock.json" or "npm-shrinkwrap.json" now that we use Yarn.

Reply:
I didn't think it was necessary